### PR TITLE
Residential Exemptions in MHR table

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* 
- bcgov/entity#17421

*Description of changes:*
- Updates to MHR Table after filing Res Exemptions
- Add unit tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1584-c0u4dv1t.web.app

**Review by doug-lovett — APPROVED** [review]
LGTM.

File: ppr-ui/src/components/tables/common/TableRow.vue [hunk 1]
```diff
@@ -398,13 +409,17 @@
...
                 <v-list-item-subtitle>
                   <img alt="exemption-icon" class="exemption-icon" src="@/assets/svgs/ic_exemption.svg" />
                   <span class="ml-1">Non-Residential Exemption</span>
                 </v-list-item-subtitle>
               </v-list-item>
-              <v-list-item @click="handleAction(item, TableActions.REMOVE)">
+              <v-list-item
+                @click="handleAction(item, TableActions.REMOVE)"
+                data-test-id="remove-mhr-row-btn"
+              >
```
Comment (cameron-eyds) [inline ppr-ui/src/components/tables/common/TableRow.vue:425]: We will have some conflicts here, looks like several changes in this area overlap.

**Review by cameron-eyds — APPROVED** [review]
Looks good to me! 
That being said, whomever goes in 2nd will have some conflicts to resolve in the TableRow. 
Have a look at my TableRow changes and let me know what you think, i'm good with either or 👍

File: ppr-ui/src/components/tables/common/TableRow.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/components/tables/common/TableRow.vue:425]: I can resolve, as your PR is bigger and should go first.

**Review by RuoxuanPengBC — APPROVED** [review]
looks great

**Comment by dimak1** [comment]
@cameron-eyds sync'ed with main, resolved conflicts... if you want to take another quick look.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1584-c0u4dv1t.web.app

File: ppr-ui/src/components/tables/common/TableRow.vue [hunk 2]
```diff
@@ -387,9 +387,21 @@
...
+                <v-list-item-subtitle>
+                  <img alt="exemption-icon" class="ml-0 exemption-icon" src="@/assets/svgs/ic_exemption.svg" />
+                  <span class="ml-1">Rescind Exemption Order</span>
+                </v-list-item-subtitle>
+              </v-list-item>
+              <v-list-item
+                v-if="isExemptionEnabled && !hasChildResExemption(item) &&
                   ![HomeLocationTypes.HOME_PARK, HomeLocationTypes.LOT].includes(item.locationType)"
                 @click="openExemption(UnitNoteDocTypes.RESIDENTIAL_EXEMPTION_ORDER, item)"
+                data-test-id="res-exemption-btn"
```
Comment (cameron-eyds) [inline ppr-ui/src/components/tables/common/TableRow.vue:404]: Looks good to me!

**Review by cameron-eyds — APPROVED** [review]
Still looks good :)

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1584-c0u4dv1t.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1584-c0u4dv1t.web.app

File: ppr-ui/src/components/tables/common/TableRow.vue [hunk 3]
```diff
@@ -820,14 +820,14 @@ export default defineComponent({
           const today = new Date()
           const expireDate = new Date()
           // expireDate.setDate(expireDate.getDate() + days)
-          var dateExpiry = moment(new Date(
+          var dateExpiry = moment.utc(new Date(
```
Comment (dimak1) [inline ppr-ui/src/components/tables/common/TableRow.vue:823]: This is the change I made (+ unit test update) for fix the dates issue. @cameron-eyds

Comment (cameron-eyds) [reply ppr-ui/src/components/tables/common/TableRow.vue:823]: I'm not fully familiar with moment or the implications but if you are confident than so am i :)

**Review by cameron-eyds — APPROVED** [review]

File: ppr-ui/src/components/tables/common/TableRow.vue (hunk 3 above)
Comment (dimak1) [reply ppr-ui/src/components/tables/common/TableRow.vue:823]: I verified that before and after this change, the number of days to expire was the same. Also, we are using UTC format in the `Date()` inside the moment function.

**Pull request merged**